src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -386,6 +380,18 @@ public bool IsValidValue(Schema schema, GraphType type, object input)
 
             if (type is ObjectGraphType || type is InputObjectGraphType)
             {
+                if (input is KeyValuePair<string, object>)
```

Comment:
Actually, I moved this out already (and into the input processor), so this is not needed. Will remove.